ios/Sources/TimetableFeature/List/TimetableLoaded.swift

```diff
@@ -6,11 +7,15 @@ import SwiftUI
 public struct TimetableLoadedState: Equatable {
     public var timetableItems: [AnyTimetableItem]
     public var selectedType: SelectedType
-    // TODO: Replace with detail state
-    public var detail: AnyTimetableItem?
+    public var selectedTimetable: AnyTimetableItem?
+    public var isSheetPresented: URL?
```

Comment:
This should be placed in `TimetableDetailState` (Currently not existing)
Timetable(List)Screen doesn't present sheet (only setting it missed..).